config.json

```diff
@@ -0,0 +1,83 @@
+{
+  "settings": {
+    "Project": {
+      "description": "Project where the commands are going to be applied.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanArtifacts": {
+      "description": "MsBuild target that deletes the artifacts output directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackages": {
+      "description": "MsBuild target that deletes the repo-local nuget package directory.",
```

Comment:
Anything under artifacts will be removed via `CleanArtifacts`; the `CleanPackages` is meant to clean the root folder nuget cache. Eventually we will need to consolidate all of our packages to one folder and look there for all packages (due to requirements I spoke of offline) so this is the start to that